docs/limitations.md

```diff
@@ -11,3 +11,12 @@ Ref: https://kubernetes.io/docs/concepts/storage/volumes/#csi
 
 > The `csi` volume type does not support direct reference from Pod and may
 > only be referenced in a Pod via a `PersistentVolumeClaim` object. 
+
+Pod without PVC
+---------------
+
+TopoLVM expects that PVCs are created in advance of their Pods.
+However, the TopoLVM webhook does not prevent the creation of Pods before their PVCs. This is because such usages are valid in other StorageClasses and the webhook cannot identify the StorageClasses without PVCs.
```

Comment:
```suggestion
However, the TopoLVM webhook does not block the creation of a Pod when there are missing PVCs for the Pod. This is because such usages are valid in other StorageClasses and the webhook cannot identify the StorageClasses without PVCs.
```